apps/web/hooks/useLabels.ts

```
@@ -1,8 +1,8 @@
 import { useMemo } from "react";
 import useSWR from "swr";
 import type { LabelsResponse } from "@/app/api/google/labels/route";
-import type { gmail_v1 } from "@googleapis/gmail";
-import { labelVisibility } from "@/utils/gmail/constants";
+import { useAccount } from "@/providers/EmailAccountProvider";
+import { EmailLabel } from "@/providers/EmailProvider";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)